### PR TITLE
Added DDP example 

### PR DESCRIPTION
**Pull request by AlekseySh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -326,6 +326,59 @@ trainer.fit(pl_model, train_dataloaders=train_loader, val_dataloaders=val_loader
...
+metric_callback = MetricValCallbackDDP(metric=EmbeddingMetricsDDP())  # ddp specific
+
+# run
+pl_model = RetrievalModuleDDP(model=model, criterion=criterion, optimizer=optimizer,
+                              loaders_train=train_loader, loaders_val=val_loader  # ddp specific
+                              )
+trainer = pl.Trainer(max_epochs=1, callbacks=[metric_callback], num_sanity_val_steps=0,
+                     accelerator="auto", devices=2, strategy=pl.plugins.DDPPlugin(), replace_sampler_ddp=False  # ddp specific
+                     )
+trainer.fit(pl_model)
```
Comment (DaloroAT) [inline README.md:376]: say we remove loaders from here

File: README.md [hunk 2]
```diff
@@ -326,6 +326,59 @@ trainer.fit(pl_model, train_dataloaders=train_loader, val_dataloaders=val_loader
...
+val_dataset = DatasetQueryGallery(df_val, dataset_root=dataset_root)
+val_loader = torch.utils.data.DataLoader(val_dataset, batch_size=4)
+metric_callback = MetricValCallbackDDP(metric=EmbeddingMetricsDDP())  # ddp specific
+
+# run
+pl_model = RetrievalModuleDDP(model=model, criterion=criterion, optimizer=optimizer,
+                              loaders_train=train_loader, loaders_val=val_loader  # ddp specific
+                              )
+trainer = pl.Trainer(max_epochs=1, callbacks=[metric_callback], num_sanity_val_steps=0,
+                     accelerator="auto", devices=2, strategy=pl.plugins.DDPPlugin(), replace_sampler_ddp=False  # ddp specific
```
Comment (DaloroAT) [inline README.md:374]: maybe group new parameters to some dict? To highlight exactly new parameters

**Review by DaloroAT — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AlekseySh) [reply README.md:374]: they are grouped in a new line already

File: README.md (hunk 1 above)
Comment (AlekseySh) [reply README.md:376]: ok

File: README.md (hunk 2 above)
Comment (AlekseySh) [reply README.md:374]: but okay

**Pull request merged**